src/utils/versionCompare.ts

```diff
@@ -1,13 +1,23 @@
 /**
  * Checks if the given version is at least the given minimum version. Expects the version numbers to be
  * dot-separated numbers, e.g. "1.0.15". String parts, such as 'preview', 'alpha', 'rc' are not supported.
+ *
+ * The allowZeroInLast parameter is used to allow the last part of the version to be zero. This is useful
+ * when running the backend with project references, as the build number is set to zero in that case.
  */
-export function isAtLeastVersion(version: string, minVersion: string): boolean {
+export function isAtLeastVersion(version: string, minVersion: string, allowZeroInLast = false): boolean {
```

Comment:
Could the third parameter here be changed to an object with options? When I came across this code   `isAtLeastVersion(altinnNugetVersion, MINIMUM_APPLICATION_VERSION.build, true)`  in `ApplicationMetadataProvider.tsx` I was wondering what the `true` value was. If instead it was called like this `isAtLeastVersion(altinnNugetVersion, MINIMUM_APPLICATION_VERSION.build, { allowZeroInLast: true })`, it would be clearer :sweat_smile:

Reply:
Good call! I made all props into an object now. I think that looks even better than `prop, prop, optionsObject?`. :partying_face:

Reply:
Even better 🤩